packages/nimble-components/src/wafer-map/modules/hover-handler.ts

```diff
@@ -0,0 +1,79 @@
+import type { WaferMap } from '..';
+import { PointCoordinates, WaferMapQuadrant } from '../types';
+
+/**
+ * HoverHandler deals with user interactions and events like hovering
+ */
+export class HoverHandler {
```

Comment:
I noticed a minor bug when playing with hover in [the storybook for this PR](https://60e89457a987cf003efc0a5b-ygmxguopok.chromatic.com/?path=/story/wafermap--wafer-map&args=dies:goodDies100). If you slowly move off of a die into the blank space between dies, sometimes the neighboring die is hovered before it should be. This is easier to see if you zoom in to a wafer map with a lot of dies. I don't see it on the default unzoomed data set.

(tested on macOS Firefox)


https://user-images.githubusercontent.com/10500124/221622488-b01084f4-79af-4c7b-af85-e5a0e6439daf.mov

Reply:
I still see this behavior in the latest storybook build but the comment is closed with no explanation. Could you provide more context about what you expect to happen with this?

Reply:
Sounds good. I'll let you decide the best way to track it in your backlog, just don't want it to be forgotten.